### DISCUSSION TITLE
AttributeError: 'MetdataProcessorDialog' object has no attribute 'comboBox_xcmd'. Did you mean: 'comboBox_xsmd'?

### DISCUSSION THREAD

**Discussion by hhh321747** [UMEP-dev/UMEP#668] [Q&A]
![bfb8f0c25da384e4710d9507e0afa7c](https://github.com/user-attachments/assets/c0d5f0a0-ce7f-4280-9811-330d04a36a43)

![89ecfd77a8af50a689846e54e2611d8](https://github.com/user-attachments/assets
![fae2e7b5b3853950a4074a297727db1](https://github.com/user-attachments/assets/d9b49c66-4938-4f78-a337-f4cde267f8a9)
/d22baac9-cfdb-4417-a7c1-714275218eab)

**Comment by biglimp** [comment]
You actually found a bug. I will transform this into an issue and fix. /F
